src/views/Home/RecentActivity/index.tsx

```diff
@@ -52,12 +51,22 @@ function RecentActivities() {
             heading={_ts('recentActivity', 'recentActivitiesHeading')}
         >
             <Card className={styles.content}>
-                <List
-                    data={recentActivitiesResponse?.results}
-                    renderer={ActivityItem}
-                    keySelector={keySelector}
-                    rendererParams={activityRendererParams}
-                />
+                {(recentActivitiesResponse?.results ?? []).length > 0 ? (
+                    <List
+                        data={recentActivitiesResponse?.results}
+                        renderer={ActivityItem}
+                        keySelector={keySelector}
+                        rendererParams={activityRendererParams}
+                    />
+                ) : (
+                    <div className={styles.emptyMessage}>
+                        <GiShrug className={styles.icon} />
+                        <div className={styles.text}>
+                            {/* FIXME: use strings with appropriate wording */}
+                            You do not have any recent activities
+                        </div>
+                    </div>
+                )}
```

Comment:
Shouldn't we also check for loading state.

Maybe introduce props:
- loading
- emptyElement
- loadingElement

inside list?